docs/guides/advanced/schemas.md

```diff
@@ -531,6 +531,25 @@ In your sync formula you only need to populate the fields of the reference objec
     Reference schemas are only resolved to rows when they are used in a sync table. If used in a formula or column format they will always appear in a broken state, even if the row they are referencing is present.
 
 
+### Column IDs
+
+In addition to a user-visible name, each column in a sync table has a stable identifier. Although this ID isn't used today, in the future it will enable advanced features like being able to rename a column without impacting existing docs. By default a property's key in the schema will be used as the column ID.
+
+In most cases this default behavior is sufficient, but in rare cases you may want to set a different ID value for the column. For example, if you generate your schema dynamically to include custom fields returned by the API, you should use the ID of those custom fields instead of their name.
```

Comment:
Nit: what do you think about "an API" here instead of "the API"?

Reply:
Done